me.drewol.Unnamed-SDVX-Clone.yml

```diff
@@ -0,0 +1,53 @@
+id: me.drewol.Unnamed-SDVX-Clone
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: usc-game
+finish-args:
+  # X11 + XShm access
+  - --share=ipc
+  - --socket=fallback-x11
+  # Wayland access
+  - --socket=wayland
+  # GPU acceleration
+  - --device=dri
+  # Needs to talk to the network
+  - --share=network
+  # Sound output
+  - --socket=pulseaudio
+modules:
+  - name: unnamed-sdvx-clone-app
+    buildsystem: simple
+    sources:
+      - type: file
+        path: me.drewol.Unnamed-SDVX-Clone.desktop
+      - type: file
+        path: me.drewol.Unnamed-SDVX-Clone.metainfo.xml
+    build-commands:
+      - install -D me.drewol.Unnamed-SDVX-Clone.metainfo.xml /app/share/metainfo/me.drewol.Unnamed-SDVX-Clone.metainfo.xml 
+      - install -D me.drewol.Unnamed-SDVX-Clone.desktop /app/share/applications/me.drewol.Unnamed-SDVX-Clone.desktop 
+  - name: unnamed-sdvx-clone
+    buildsystem: cmake-ninja
+    sources:
+      - type: git
+        url: https://github.com/zlib-ng/zlib-ng
+        tag: 2.1.3
+        dest: flatpak-deps/zlib-ng/
+      - type: archive
+        url: https://github.com/curl/curl/releases/download/curl-7_75_0/curl-7.75.0.tar.xz
```

Comment:
curl is in the runtime.

Reply:
I downgraded the runtime in order to get the correct version of curl this builds against, and added sdl2 so that window decorations work correctly on wayland. Is this a good way to solve this?

Reply:
definitely not since 21.08 is EOL.

Reply:
ok, in that case what would you recommend? I don't want to ask the maintainer to upgrade curl if I can help it.

Reply:
I'd suggest updating it as soon as possible. curl is already in runtime and a lot of components depend on it. Who knows what will happen with an outdated libcurl installed to `/app`.

And of top of that it receives a bunch of security fixes every release.